scripts/ci/tags.yaml

```diff
@@ -804,3 +804,49 @@ ci_applications_nrf_desktop:
     - zephyr/subsys/settings/
     - zephyr/subsys/storage/
     - zephyr/subsys/usb/
+
+ci_tests_subsys_caf:
+  files:
+    - nrf/subsys/app_event_manager/
+    - nrf/subsys/caf/
+    - nrf/tests/subsys/caf/
+
+ci_tests_subsys_partition_manager:
+  files:
+    - nrf/subsys/partition_manager/
+    - nrf/tests/subsys/partition_manager/
+
+ci_tests_subsys_emds:
+  files:
+    - nrf/subsys/bluetooth/
+    - nrf/subsys/emds/
+    - nrf/subsys/mpsl/
+    - nrf/subsys/nrf_security/
+    - nrf/tests/subsys/emds/
+    - zephyr/subsys/bluetooth/
+
+ci_tests_lib_nrf_fuel_gauge:
+  files:
+    - nrf/tests/lib/nrf_fuel_gauge/
+    - nrfxlib/nrf_fuel_gauge/
+
+ci_tests_lib_at_cmd_parser:
```

Comment:
`at_cmd_parser` is going to be deprecated for 2.8.0 and eventually removed but I guess it should be here until then. Not sure anybody will remove it here although it doesn't break anything.

Reply:
Let's keep it until test is in code.
Nothing will happen when code will be remove and tag will stay.